soh/src/overlays/gamestates/ovl_file_choose/z_file_choose.c

```diff
@@ -2656,6 +2711,9 @@ void FileChoose_Init(GameState* thisx) {
     DmaMgr_SendRequest1(this->parameterSegment, (u32)_parameter_staticSegmentRomStart, size, __FILE__,
                         __LINE__);
 
+    // Load some registers used by the dialog system
+    func_80112098(NULL); // Passing in NULL as we dont have a playstate, and it isn't used in the func
```

Comment:
might be nice to rename this method to not just be a `func_`, but if nobody has good name ideas that doesn't need to block this

Reply:
This seems to basically set all register values to defaults (its ran during `Play_Init`)
(or rather this function is just a wrapper for another function that actually sets the register values)

Maybe `Registers_Init` ?

Reply:
Looks Like decomp documented it as `Regs_InitData` so using that instead.